composer.json

```diff
@@ -2,16 +2,15 @@
     "name": "spaggel/tooltip",
     "description": "Product attribute tooltips module for Magento 2",
     "require": {
-        "magento/framework": "~102.0",
-        "magento/module-backend": "~101.0",
-        "magento/module-catalog": "~103.0",
+        "magento/framework": "~103.0",
+        "magento/module-backend": "~102.1",
```

Comment:
Where did you get these dependencies from? Magento 2.4 has `magento/module-backend` in version `102.0.1`. Hence, by doing this, we only support `2.4.1`. Even though people should use `2.4.1` already, I do not see the point of restricting this so much.